<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEditor</class>
 <widget class="QWidget" name="DataEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRXP Data Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="3">
      <widget class="QPushButton" name="button_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>192</width>
            <height>492</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QListWidget" name="central_menu">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Actors</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Classes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Skills</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Items</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Weapons</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Armors</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Enemies</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Troops</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>States</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Animations</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tilesets</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Common Events</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>System</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Scripts</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>252</red>
                 <green>252</green>
                 <blue>252</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>252</red>
                 <green>252</green>
                 <blue>252</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>160</red>
                 <green>162</green>
                 <blue>162</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>255</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>251</width>
              <height>400</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QListWidget" name="object_list"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Change Maximum</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>7</number>
         </property>
         <widget class="EditActors" name="page_actors"/>
         <widget class="EditClasses" name="page_classes"/>
         <widget class="EditSkills" name="page_skills"/>
         <widget class="EditItems" name="page_items"/>
         <widget class="EditWeapons" name="page_weapons"/>
         <widget class="EditArmors" name="page_armors"/>
         <widget class="EditEnemies" name="page_enemies"/>
         <widget class="EditTroops" name="page_troops"/>
         <widget class="QWidget" name="page_states"/>
         <widget class="QWidget" name="page_animations"/>
         <widget class="QWidget" name="page_tilesets"/>
         <widget class="QWidget" name="page_commonevents"/>
         <widget class="QWidget" name="page_system"/>
         <widget class="QWidget" name="page_scripts"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditActors</class>
   <extends>QWidget</extends>
   <header>editors/actors/editactors.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditClasses</class>
   <extends>QWidget</extends>
   <header>editors/classes/editclasses.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditSkills</class>
   <extends>QWidget</extends>
   <header>editors/skills/editskills.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditItems</class>
   <extends>QWidget</extends>
   <header>editors/items/edititems.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditWeapons</class>
   <extends>QWidget</extends>
   <header>editors/weapons/editweapons.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditArmors</class>
   <extends>QWidget</extends>
   <header>editors/armors/editarmors.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditEnemies</class>
   <extends>QWidget</extends>
   <header>editors/enemies/editenemies.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditTroops</class>
   <extends>QWidget</extends>
   <header>editors/troops/edittroops.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
